src/renderer/extensions/vueNodes/components/ImagePreview.vue

```diff
@@ -186,8 +187,29 @@ const handleImageError = () => {
   actualDimensions.value = null
 }
 
-const handleEditMask = () => {
-  void commandStore.execute('Comfy.MaskEditor.OpenMaskEditor')
+const handleEditMask = async () => {
+  if (props.nodeId) {
+    const node = app.rootGraph?.getNodeById(props.nodeId)
+    if (node) {
+      node.imageIndex = currentIndex.value
+
+      node.imgs = await Promise.all(
+        props.imageUrls.map((url) => {
+          return new Promise<HTMLImageElement>((resolve) => {
+            const img = new Image()
+
+            img.onload = () => resolve(img)
+
+            img.src = url
```

Comment:
@christian-byrne Is this going to run into the same CORS issue you were working on fixing for Cloud?

Reply:
added